### PR TITLE
LPS-166416 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LPS-166416

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

File: modules/apps/address/address-test/src/testFunctional/tests/CountriesManagement.testcase [hunk 1]
```diff
@@ -458,13 +458,13 @@ definition {
...
+			AssertTextEquals(
+				locator1 = "Message#REQUIRED_INFO",
+				value1 = "The Key field is required.");
 
-			AssertElementPresent(
-				key_fieldFieldLabel = "Region Code",
-				locator1 = "Message#ERROR_FIELD_REQUIRED");
+			AssertTextEquals(
+				locator1 = "Message#REQUIRED_INFO",
+				value1 = "The Region Code field is required.");
```
Comment (patriciajeanperez) [inline modules/apps/address/address-test/src/testFunctional/tests/CountriesManagement.testcase:467]: @melodywu Did we test if this works? I see there are two error messages you are asserting. Is the path smart enough to assert both error messages?

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/UsersAdmin.testcase [hunk 2]
```diff
@@ -2296,7 +2296,9 @@ definition {
 		Button.clickSave();
 
 		for (var fieldFieldLabel : list "Screen Name,Email Address,First Name,Last Name") {
-			Alert.viewSpecificRequiredField(fieldFieldLabel = "${fieldFieldLabel}");
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/UsersAdmin.testcase:2299]: @melodywu This macro right here might be able to assert a specific error message based on the field. You can try to update this macro and the path in here to be able to do what you want and see if that works.

**Review by patriciajeanperez — COMMENTED** [review]
Hi @melodywu, please review and make sure all the tests you update pass before you send me a PR. Don't forget to assign the JIRA ticket to me and move it to In Review when it is ready. Thank you.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/UsersAdmin.testcase (hunk 2 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/UsersAdmin.testcase:2299]: ok

**Pull request closed without merging**